src/v2/Apps/Order/Components/SavedAddressItem.tsx

```diff
@@ -39,13 +39,32 @@ export const SavedAddressItem: React.FC<SavedAddressItemProps> = (
           nameAndAddressLine.map(
             (line: string, index: number) =>
               line && (
-                <Text
-                  style={{ textTransform: "capitalize" }}
-                  variant="text"
-                  key={index}
-                >
-                  {line}
-                </Text>
+                <Flex key={index} justifyContent="row" alignItems="center">
+                  <Text
+                    style={{ textTransform: "capitalize" }}
```

Comment:
We can use the `styled-system` prop `textTransform='uppercase'` here

Reply:
@Serge0n mind making this change? Thanks!

Reply:
@sweir27 Done)